src/components/Modals/FiatRamps/config.ts

```diff
@@ -68,12 +72,14 @@ export const supportedFiatRamps: SupportedFiatRamp = {
       }
     },
     isImplemented: true,
+    isActive: true,
```

Comment:
I suppose we could also reuse the `isImplemented` value here

Reply:
@lmyslinski `isImplemented` is effectively used *alongside* (not instead of) feature flagging to define whether or not a fiat ramp is currently available for use (`true`), or if it's "Coming Soon" (`false`).

If we simply want to hide a fiat ramp during development, we can feature flag it, but the way we usually go with is both feature flagging it and having `isImplemented: false` initially, then when the implementation is fully ready, set it to `isImplemented: true` so it can be tested with the flag on, and eventually toggle off/remove the flag.

IMO this added option makes sense, the way we've managed feature detection previously was quite hacky